src/Benchmarks/Program.cs

```diff
@@ -133,6 +140,21 @@ private static int GetThreadCount(IConfigurationRoot config)
             var threadCountValue = config["threadCount"];
             return threadCountValue == null ? -1 : int.Parse(threadCountValue);
         }
+
+        private static IConnectionFilter GetConnectionFilter(IConfigurationRoot config, IConnectionFilter prevFilter)
+        {
+            var connectionFilterValue = config["connectionFilter"];
+            if (string.IsNullOrEmpty(connectionFilterValue))
+            {
+                return null;
+            }
+            else
+            {
+                var connectionFilterType = Type.GetType(connectionFilterValue, throwOnError: true);
```

Comment:
The problem is that those connection filters are non trivial to create as they have arguments. You'll end up hard coding the construction of specific filters. For example, logging https://github.com/aspnet/KestrelHttpServer/blob/dev/src/Microsoft.AspNetCore.Server.Kestrel/Filter/LoggingConnectionFilter.cs, you'd have a pass an `ILogger` and for SSL https://github.com/aspnet/KestrelHttpServer/blob/dev/src/Microsoft.AspNetCore.Server.Kestrel.Https/HttpsConnectionFilter.cs, you need to pass options. That's why the extension methods exist.